src/server/cea_608_encoder/scene_utility.py

```diff
@@ -1,13 +1,41 @@
 import src.server.cea_608_encoder.caption_string_utility as utils
 
-def create_bytes_for_scene_background_color(color: str):
-    pass
+background_colors = {"white": 0x0, "green": 0x2, "blue": 0x4, "cyan": 0x6, 
+    "red": 0x8, "yellow": 0xa, "magenta": 0xc, "black": 0xe}
+
+def create_bytes_for_scene_background_color(color: str, transparency = False):
+    byte_list = []
+    first_byte = 0x00
+    second_byte = 0x00
+
+    color = color.lower()
+    if color in background_colors:
+        first_byte = 0x10
+        second_byte = 0x20 + background_colors[color]
+        if transparency == True: 
+            second_byte += 0x1
+    else:                           # if color not found, then no background
+        first_byte = 0x17
```

Comment:
if this is what we default to when there is not color in the background colors dictionary, do we need to initialize them to 0x00 on line 8, 9 or could these values replace those.

Reply:
Removed

Reply:
I was thinking you move these bytes outside the else and remove the else altogether. This looks a little clunky because most programming languages the `first_byte` and `second_byte` would be scoped only within the if/else block since it's not declared outside.

python let's you get away with how the assignment is happening here but if you wrote this same code in c++ when you try to access first_byte or second_byte in say lines 19-21 you would get an error saying they aren't defined because they would only be scoped inside the if/else on lines 10-17.

I would remove the else condition and move the default values for first_byte, second_byte up to where you declare byte_list

Reply:
Done